.github/workflows/main.yml

```diff
@@ -8,9 +8,9 @@ concurrency: Testing
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
-    branches: [ master, dev ]
+    branches: [ master, dev, prod21 ]
```

Comment:
Do _master_ and _dev_ need here

Reply:
In all other branches, the workflow looks like this. Suggesting to leave only prod21?

Reply:
I have tried to make the pull request if only prod21 is present and it has worked properly.
But I know nothing about side effects that is actually why I asked if those branches need to be in main.yml.
In my opinion, they do not necessarily need to be in this file but my opinion can be wrong.